crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -3063,6 +3063,11 @@ type ServiceConfig {
 	"""
 	maxPageSize: Int!
 	"""
+	Maximum time in milliseconds that it will wait to get the results from the execution. Note,
+	that the transaction might be still queued for execution beyond this timeout.
```

Comment:
noob q: what does this mean? like, it's already submitted to validator, and we timeout waiting for execution result?

Reply:
The results of this execution will not be returned if this times out, even though the execution will continue and probably be finalized (if it goes beyond these 60s). 
Would love to have a clearer phrasing, so let me know if you know a better one.